src/components/text-input/index.md

```diff
@@ -116,6 +116,20 @@ Do not set the `inputmode` attribute to `decimal` as it causes some devices to b
 
 Do not use `<input type="number">` unless your user research shows that there’s a need for it. With `<input type="number">` there’s a risk of users accidentally incrementing a number when they’re trying to do something else - for example, scroll up or down the page. And if the user tries to enter something that’s not a number, there’s no explicit feedback about what they’re doing wrong.
 
+### Codes and sequences
+
+If you're asking the user to enter a code or sequence that they're unlikely to have memorised - such as an application reference ID, account number or security code - style the input's text to visually separate each character. This makes it easier for the user to visually check that the code they've typed is correct.
```

Comment:
```suggestion
To make it easier for the user to visually check that the code they've typed is correct, you must style the input's text to visually separate each character. This is important if you're asking the user to enter a code or sequence they're unlikely to have memorised, such as an application reference ID, account number or security code. 
```

Reply:
Yeah, I like the approach. For the first sentence, we could match the a bit of a pattern this page has developed and say:

> Help the user visually check that the code they've typed is correct by styling the input's text to separate each character.

or 

> Style the input's text to separate each character to help the user visually check the code they've typed in is correct.

Reply:
@calvin-lau-sig7 I've amended my suggestion with your first option.